esp/esp/program/forms.py

```diff
@@ -62,23 +61,64 @@ class ProgramCreationForm(BetterModelForm):
+        self.program_module_ids = [[x.id for x in ProgramModule.objects.filter(admin_title__in=['Student Optional Fees', 'Accounting', 'Credit Card Payment Module (Stripe)', 'Financial Aid Application',                                                                                        'Easily Approve Financial Aid Requests'])],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title__in=['Accounting', 'Credit Card Payment Module (Stripe)', 'Financial Aid Application',
+                                                                                                'Easily Approve Financial Aid Requests'])],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title='Teacher Logistics Quiz')],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title__in=['Teacher Surveys', 'Survey Management'])],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title__in=['Student Surveys', 'Survey Management'])],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title='Teacher Custom Form')],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title='Student Custom Form')],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title='Student Lunch Period Selection')],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title='Donation Module')],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title__in=['Teacher Training and Interview Signups', 'Manage Teacher Training and Interviews'])],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title__in=['Two-Phase Student Registration', 'Lottery Frontend'])],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title__in=['Student Registration Phase Zero', 'Manage Student Registration Phase Zero'])],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title__in=['Application Review for Admin', 'Admin Admissions Dashboard'])],
+                                   [x.id for x in ProgramModule.objects.filter(admin_title__in=['Teacher Admissions Dashboard', 'Application Reviews for Teachers', 'Application Review for Admin', 'Admin Admissions Dashboard'])]
+                                   ]
+        self.program_modules_questions = ['Will you have extra costs (shirts or lunch)?', # Accounting, Student Optional Fees, Credit Card Payment Module (Stripe), Financial Aid Application, Easily Approve Financial Aid Requests
+                                          'Will you charge for the program?', # Accounting, Credit Card Payment Module (Stripe), Financial Aid Application, Easily Approve Financial Aid Requests
```

Comment:
I would suggest having the credit card modules be a separate question (e.g. "Will you be accepting credit card payments"), since not all programs accept (or are allowed to accept) credit card payments and we don't want to add this module just because they are charging for a program.

Reply:
Yeah, I get that. I think my concern was that that allows people to select the credit card box without selecting either the "charge for the program" or "charge for extra stuff" box, which doesn't make sense. But I guess there's no real problem if they do that, right?

Reply:
I think that'd be fine.

Reply:
Cool, will do that then!

Reply:
fixed in commit 419ec54